src/main/java/binnie/core/triggers/TriggerWorking.java

```diff
@@ -1,43 +0,0 @@
-package binnie.core.triggers;
-
-import binnie.core.machines.Machine;
-import binnie.core.machines.power.IProcess;
-
-// TODO unusing class?
-public class TriggerWorking {
```

Comment:
I think these triggers are supposed to be used by BuildCraft logic gates, are you sure they do not work?